changelogs/fragments/gcp-facts-info-rename.yaml

```
@@ -0,0 +1,53 @@
+minor_changes:
+- The ``gcp_bigquery_dataset_facts`` module was renamed to :ref:`gcp_bigquery_dataset_info <gcp_bigquery_dataset_info_module>`.
```

```suggestion
- The ``gcp_bigquery_dataset_facts`` module was renamed to ``gcp_bigquery_dataset``.
```
Sorry, one more. Since the changelog is not part of the docsite, ReST links won't work.

---

no problem! made the change on all of the changelog fragments.